frontend/src/scenes/surveys/SurveySettings.tsx

```diff
@@ -38,6 +38,8 @@ function SurveyPopupToggle(): JSX.Element {
                 labelClassName="text-base font-semibold"
                 checked={!!currentTeam?.surveys_opt_in}
                 className="p-0"
+                disabled={currentTeamLoading}
```

Comment:
can it be that `currentTeamLoading` is loaded but errored so `currentTeam` is still undefined?